src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public static void doWork() {
+        System.out.println("Excavator started its work ");;
```

Comment:
The message in `System.out.println` should be more informative. It should indicate that it is the `Excavator` that started its work. For example: `System.out.println("Excavator is working");`